### PR TITLE
Ft create a party endpoint 163519076

### PR DESCRIPTION
**Pull request by fantastic-genius**
#### What does this PR do?
To add endpoint used for creating a party
#### Description of Task to be completed?
Wrote function lying in controller to create new party
Wrote a function in middleware to validate the values being posted
Created its route within the route file
#### How should this be manually tested?
None
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
163519076
#### Screenshots (if appropriate)
None
#### Questions:
None

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    "extends": "airbnb-base"
+};
```
Comment (veeqtor) [inline .eslintrc.js:3]: Add a new line to the end of your files

File: controller/partyController.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import db from "../db/db"
```
Comment (veeqtor) [inline controller/partyController.js:1]: Nice work @fantastic-genius , However,  I can't find where you are persisting the party you created into the  data structure database. Please do that, thanks.

**Review by veeqtor — CHANGES_REQUESTED** [review]

File: .eslintrc.js (hunk 1 above)
Comment (fantastic-genius) [reply .eslintrc.js:3]: do u meant adding an additional line to the page you highlighted to make the lines 4 without anything in that line?
and if so, i will like to know why?

File: controller/partyController.js (hunk 2 above)
Comment (fantastic-genius) [reply controller/partyController.js:1]: Also, I don't understand the data structure persistence, as i have done some little search on it but couldn't understand how to go about such. I will appreciate if you could help out with resources on it. However, in the challenge 2 requirement as contained in what was sent across to us, we are to use data structures without persistent (edited)

File: .eslintrc.js (hunk 1 above)
Comment (veeqtor) [reply .eslintrc.js:3]: Yeah, you can look through this resource to better understand why. https://evanhahn.com/newline-necessary-at-the-end-of-javascript-files/

File: controller/partyController.js (hunk 2 above)
Comment (veeqtor) [reply controller/partyController.js:1]: I can see that you have done that on this P.R #46

File: .eslintrc.js (hunk 1 above)
Comment (fantastic-genius) [reply .eslintrc.js:3]: ok. Thanks. I do appreciate

File: controller/partyController.js (hunk 2 above)
Comment (fantastic-genius) [reply controller/partyController.js:1]: Ok thanks

**Pull request merged**